core/src/main/java/com/google/instrumentation/trace/ScopedSpanHandle.java

```diff
@@ -38,12 +38,12 @@
   }
 
   /**
-   * Uninstalls the {@code Span} from the current context and ends it with default options. If the
-   * span has been already ended then {@link Span#end()} is a no-op.
+   * Uninstalls the {@code Span} from the current context and ends by calling {@link Span#end()}
+   * it with default options.
```

Comment:
Thanks. also removed the "with default options" because that is what Span#end does.